engine/server/sv_client.c

```diff
@@ -2561,6 +2561,58 @@ void SV_ParseCvarValue2( sv_client_t *cl, sizebuf_t *msg )
 	Con_Reportf( "Cvar query response: name:%s, request ID %d, cvar:%s, value:%s\n", cl->name, requestID, name, value );
 }
 
+/*
+===================
+SV_ParseVoiceData
+===================
+*/
+void SV_ParseVoiceData( sv_client_t *cl, sizebuf_t *msg )
+{
+	char received[4096];
+	sv_client_t	*cur;
+	int i, client;
+	uint length, size, frames;
+
+	cl->m_bLoopback = MSG_ReadByte( msg );
+
+	frames = MSG_ReadByte( msg );
+
+	size = MSG_ReadShort( msg );
+	client = cl - svs.clients;
+
+	if ( size > sizeof( received ) )
+	{
+		Con_DPrintf( "SV_ParseVoiceData: invalid incoming packet.\n" );
+		SV_DropClient( cl, false );
+		return;
+	}
+
+	if ( !Cvar_VariableInteger( "sv_voiceenable" ) )
+		return;
+
+	MSG_ReadBytes( msg, received, size );
+
+	for( i = 0, cur = svs.clients; i < svs.maxclients; i++, cur++ )
+	{
+		if ( cur->state < cs_connected && cl != cur )
+			continue;
+		
+		length = size;
+
+		if ( MSG_GetNumBytesLeft( &cur->datagram ) < length + 6 )
```

Comment:
Wtf `length + 6` means here?

Reply:
At least, leave a comment that 6 is a number of bytes for other parts of message. :)